wiki/osu!stream/Song_List/en.md

```diff
@@ -1,5 +1,5 @@
 <!-- wikilink -->
-[osu!stream wikilink]: .,/ "osu!stream"
+[osu!stream wikilink]: /wiki/osu!stream "osu!stream"
```

Comment:
```suggestion
[osu!stream wikilink]: .. "osu!stream"
```

Reply:
gonna go the opposite way, it looks like almost nobody uses `..` so I'm just removing all of them. avoids some parsing trouble /shrug